js/post/update_counts.js

```diff
@@ -1,8 +1,14 @@
+creditCountSpec = 0;
+creditCountMaj = 0;
+creditCountMin = 0;
+creditCount300and400 = {'Spec': 0, 'Maj': 0, 'Min': 0};
+
+
 /**
  * Updates number of completed courses in Specialist.
 **/
 function updateCompletedSpecCourses () {
-    'use-strict';
+    'use-strict'
```

Comment:
Oh yeah, it definitely is- I didn't mean to imply that it didn't! However, I can see now how it can be read that way- sorry! Thanks for the references, and thanks for asking for clarification!